.circleci/config.yml

```diff
@@ -0,0 +1,170 @@
+          yq write --inplace deployments/webserver.yml spec.template.spec.hostAliases[0].hostnames[3] $SECOND_DOMAIN_HOSTNAME
+        working_directory: project-base/k8s
+
+    - run:
+        name: Add persistent volumes into webserver and postgres manifest
+        command: |
+          yq write --inplace deployments/webserver.yml spec.template.spec.volumes[0].hostPath.path /workspace/shopsys/$CIRCLE_BRANCH/source
+          yq write --inplace deployments/postgres.yml spec.template.spec.volumes[0].hostPath.path /workspace/shopsys/$CIRCLE_BRANCH/db
+        working_directory: project-base/k8s
+
+    - run:
+        name: Set commit hash for webserver and postgres to redeploy with volumes set
+        command: |
+          yq write --inplace deployments/postgres.yml spec.template.metadata.labels.commit $CIRCLE_SHA1
+          yq write --inplace deployments/postgres.yml spec.selector.matchLabels.commit $CIRCLE_SHA1
+          yq write --inplace deployments/webserver.yml spec.template.metadata.labels.commit $CIRCLE_SHA1
+          yq write --inplace deployments/webserver.yml spec.selector.matchLabels.commit $CIRCLE_SHA1
+        working_directory: project-base/k8s
+
+    - run:
+        name: Replace docker image in deployment definition
+        command: |
+          DOCKER_IMAGE_TAG="$(cksum ../docker/php-fpm/Dockerfile | cut -d ' ' -f1)"
+          yq write --inplace deployments/webserver.yml spec.template.spec.containers[0].image $DOCKER_USERNAME/$DOCKER_IMAGE:$DOCKER_IMAGE_TAG
+        working_directory: project-base/k8s
+
+    - run:
+        name: Create configmap manifests
+        command: |
+          kubectl create configmap nginx-configuration --from-file ../../docker/nginx/nginx.conf --dry-run --output=yaml > nginx-configuration.yml
+          kubectl create configmap php-configuration --from-file ../docker/php-fpm/php-ini-overrides.ini --dry-run --output=yaml > php-configuration.yml
+        working_directory: project-base/k8s
+
+    - run:
+        name: Setup domain a parameters files
+        command: |
+          FIRST_DOMAIN_HOSTNAME=$CIRCLE_BRANCH-circle.$DEVELOPMENT_SERVER_DOMAIN
+          SECOND_DOMAIN_HOSTNAME=2.$CIRCLE_BRANCH-circle.$DEVELOPMENT_SERVER_DOMAIN
+          # Copy files from teplates
+          cp project-base/app/config/domains_urls.yml.dist project-base/app/config/domains_urls.yml
+          cp project-base/app/config/parameters_test.yml.dist project-base/app/config/parameters_test.yml
+          cp project-base/app/config/parameters.yml.dist project-base/app/config/parameters.yml
+          # Add domains into domain file
+          yq write --inplace project-base/app/config/domains_urls.yml domains_urls[0].url http://$FIRST_DOMAIN_HOSTNAME
+          yq write --inplace project-base/app/config/domains_urls.yml domains_urls[1].url http://$SECOND_DOMAIN_HOSTNAME
+
+    - run:
+        name: Set SSH credentials
+        command: |
+          echo -e $SERVER_SSL_CERT > ~/shopsys
```

Comment:
From the filename it's not apparent it's a certificate (and what for). Maybe rename it to `~/.heimdall_cert`?

Reply:
Hmm, I thought that this is configuration that we will be giving to our community. So i would ommit specific names like `heimdall`. But i agree that the name is wrong. 

We can talk about exact name later, but for now i just name it `cluster_cert`.